resources/views/layouts/navbar.blade.php

```diff
@@ -110,12 +131,18 @@ class="material-icons">menu</i></a>
                                 </a>
                             </li>
                             @endcan
+
+                            <!-- TODO can -->
```

Comment:
I think this is okay to be there for everyone involved, but currently it should be disabled as it's not working yet.